### PR TITLE
Add google-cloud-kms

### PR DESCRIPTION
**Pull request by elvirag**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/google-cloud-kms/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - google.cloud.kms_v1
+    - google.cloud.kms_v1.gapic
+    - google.cloud.kms_v1.gapic.transports
+    - google.cloud.kms_v1.proto
+
+about:
+  home: https://github.com/GoogleCloudPlatform/google-cloud-python
+  license: Apache Software
+  license_family: APACHE
+  license_file:
```
Comment (marcelotrevisani) [inline recipes/google-cloud-kms/meta.yaml]: Please, add the license

File: recipes/google-cloud-kms/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - google
+    - google.cloud
+    - google.cloud.kms_v1
+    - google.cloud.kms_v1.gapic
+    - google.cloud.kms_v1.gapic.transports
+    - google.cloud.kms_v1.proto
+
+about:
+  home: https://github.com/GoogleCloudPlatform/google-cloud-python
+  license: Apache Software
```
Comment (marcelotrevisani) [inline recipes/google-cloud-kms/meta.yaml]: that should be just ``Apache``

Comment (chrisburr) [reply recipes/google-cloud-kms/meta.yaml]: To follow the SPDX naming scheme it should be:
```suggestion
  license: Apache-2.0
```

File: recipes/google-cloud-kms/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  sha256: 2167e1c599ea1d9fed96d317ad817d0b37497460c70f11aafa13a24ede7c9c35
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - enum34   # [ py<34 ]
+    - google-api-core-grpc >=1.4.0
```
Comment (mariusvniekerk) [inline recipes/google-cloud-kms/meta.yaml]: these are probably not needed at build time

**Comment by mariusvniekerk** [comment]
```
conda.exceptions.UnsatisfiableError: The following specifications were found to be in conflict:
  - grpc-google-iam-v1[version='>=0.11.4'] -> googleapis-common-protos-grpc[version='>=1.5.9,<2.0a0'] -> googleapis-common-protos[version='>=1.6,<1.7.0a0'] -> python[version='>=3.7,<3.8.0a0']
```

**Comment by elvirag** [comment]
> ```
> conda.exceptions.UnsatisfiableError: The following specifications were found to be in conflict:
>   - grpc-google-iam-v1[version='>=0.11.4'] -> googleapis-common-protos-grpc[version='>=1.5.9,<2.0a0'] -> googleapis-common-protos[version='>=1.6,<1.7.0a0'] -> python[version='>=3.7,<3.8.0a0']
> ```

I'm not sure how to fix it, actually.

**Comment by mariusvniekerk** [comment]
you have unsatisfyable constraints on some of those google libraries

**Pull request closed without merging**